warehouse/admin/templates/admin/projects/release_detail.html

```diff
@@ -229,7 +229,15 @@ <h3>Description Rendering</h3>
         </tr>
         <tr>
           <td>Detected Content-Type</td>
-          <td><code>{{ release.description.content_type }}</code></td>
+          <td>
+            {% if release.description.content_type is none %}
+            <code>-NULL-</code>
+            {% elif release.description.content_type == "" %}
+            <code>-BLANK-</code>
+            {% else %}
+            <code>{{ release.description.content_type }}</code>
+            {% endif %}
+          </td>
```

Comment:
I realized it was tricky to determine if the content type was null or black string, so I added a carveout here for that.

I tried to use [jinja's `default()`](https://jinja.palletsprojects.com/en/3.1.x/templates/#jinja-filters.default), but got a bit lost and couldn't make it work. Happy to change if someone has a better idea!